src/main/scala/net/cakesolutions/CakeBuildPlugin.scala

```diff
@@ -13,52 +13,72 @@ import sbt.Keys._
 import sbt._
 import scoverage.ScoverageKeys._
 
+// scalastyle:off method.length
```

Comment:
Really?  If, even in this simple code, we cant get the rules to work, it questions their validity.  I guess here it must be `sensibleTestSettings` method but i dont see too much wrong with its length, its easy to read and follow.   I suggest upping the method length in the `scalastyle.conf`